infrastructure/builder/src/build/service/web-ui-nginx-site.conf

```diff
@@ -26,7 +26,7 @@ gzip_types
 server {
   listen 80;
   server_name localhost;
-  root /app/build;
+  root /app/ui/build;
 
   location / {
     try_files $uri @rewrites;
```

Comment:
The docs entry file is docs.html. We should do something similar to https://github.com/taskcluster/taskcluster/blob/32673bfefcb072fecce052d814a2d1738f5be5df/ui/.neutrinorc.js#L38-L40 so that when a user navigates to a url that starts with `/docs`, we use `docs.html` instead of the default `index.html`. I think this can be done under `location @rewrites` directive.

_Note: The redirect in `neutrinorc.js` is only for the dev environment. That's why we still need nginx to do it for us in production._

Reply:
So we already have

```
location @rewrites {
    rewrite ^(.+)$ /index.html last;
  }
```

in the nginx configuration. do I just add the docs part in there? like

```
location @rewrites {
    rewrite ^(.+)$ /index.html last;
    rewrite ^\/docs/$ /docs.html last;
  }
```

?

Reply:
and is it something that needs to be done in this PR? Like, do docs exist in tc-web?

Reply:
Yes, docs partially exist in tc-web https://taskcluster-web.netlify.com/docs. tc-web is somewhat a two single page applications (index.html and docs.html).

> do I just add the docs part in there?

Looks good to me but I haven't dabbled much with nginx in the past. You'd have to see if https://owlish.imbstack.com/docs works for you. It should look like that:

<img width="1679" alt="screen shot 2019-02-05 at 5 03 57 pm" src="https://user-images.githubusercontent.com/3766511/52307246-0e042100-2968-11e9-9823-a603c6900c93.png">

Reply:
> and is it something that needs to be done in this PR?

I guess it can wait.